libutp/utp.go

```diff
@@ -75,7 +78,7 @@ const (
 	rstInfoTimeout = 10000
 	rstInfoLimit   = 1000
 
-	// 29 seconds determined from measuring many home NAT devices
+	// 29 seconds determined from measuring many home NAT devices.
 	keepaliveInterval = 29000
```

Comment:
It might be clearer to use time.Duration as the type for time intervals... but this doesn't need to happen in this PR.